css/builder.css

```diff
@@ -556,6 +556,39 @@ input[type="time"]:not(.basic-time-picker)::-webkit-calendar-picker-indicator {
   opacity: 0;
 }
 
+.fl-date-range {
+  display: flex;
+  align-items: center;
+  margin-top: 12px;
+  flex-wrap: nowrap;
+}
+
+.fl-date-range .form-group {
+  width: 100%;
+  min-width: 156px;
+}
+
+.fl-date-range .arrow-right {
```

Comment:
The arrow should be part of the JS API styling. The form component won't need to apply further styling unless it's styling specific to the form component. We want everyone else using the JS API to have the right styles including the arrow.